test/functional/feature_notifications.py

```diff
@@ -13,13 +13,19 @@
     connect_nodes,
 )
 
+# Linux allow all characters other than \x00
+# Windows disallow control characters (0-31) and /\?%:|"<>
+FILE_CHAR_START = 32 if os.name == 'nt' else 1
+FILE_CHAR_END = 128
```

Comment:
Why 128? If we're aiming for only ASCII, it should end at 127...? (Yes, I see that the range excludes the end char, but the name for this implies inclusive)